db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20171107211226) do
+ActiveRecord::Schema.define(version: 20171107214057) do
```

Comment:
This schema has the wrong migration version right? Your latest migration is `db/migrate/20171109210835_remove_outquotes_from_articles.rb`.